### PR TITLE
Implement nav bar (#4)

### PR DESCRIPTION
**Pull request by damdiana**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,13 +2,40 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Formular</title>
```
Comment (iampava) [inline index.html]: Let's add the actual title from the Smashing magazine page.

File: index.html [hunk 2]
```diff
@@ -2,13 +2,40 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
```
Comment (iampava) [inline index.html:5]: Why did you delete these 3 lines?

File: index.html [hunk 3]
```diff
@@ -2,13 +2,40 @@
...
+            <li> <a href="https://www.smashingmagazine.com/online-workshops/" target="_blank" rel="noopener noreferrer">
+                    Workshops </a> </li>
+            <li> <a href="https://www.smashingmagazine.com/events/" target="_blank" rel="noopener noreferrer">
+                    Conferences </a> </li>
+            <li> <a href="https://www.smashingmagazine.com/membership/" target="_blank" rel="noopener noreferrer">
+                    Membership </a> </li>
+        </ul>
+        <form>
+
+            <input placeholder=" Search articles..." type="text" name="search">
```
Comment (iampava) [inline index.html:30]: Why is there an empty space at the beginning of this placeholder? Let's remove it.

File: style.css [hunk 4]
```diff
@@ -6,4 +6,97 @@
...
+
+}
+
+/* nav bar */
+
+.nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    /* justify-content: space-around; */
```
Comment (iampava) [inline style.css]: Let's remove all commented code.

File: style.css [hunk 5]
```diff
@@ -6,4 +6,97 @@
...
+body {
+    background: var(--bg-color);
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+}
+
+/* nav bar */
+
+.nav {
+    display: flex;
```
Comment (iampava) [inline style.css]: You are duplicating some rules a couple of times. Let's create utility classes for them.

For example, you can have a class named `flex` that adds `display: flex`. Another one `justify-content-between` that adds `justify-content: space-between`. Then we use these classes and get rid of duplicate code.

File: style.css [hunk 6]
```diff
@@ -6,4 +6,97 @@
...
+    text-decoration: none;
+    display: block;
+    font-size: 19px;
+    padding: 0.5rem;
+    border: transparent dotted 3px;
+
+}
+
+nav a:link {
+    color: var(--white);
```
Comment (iampava) [inline style.css:31]: Why do you have this rule 3 times?

File: style.css [hunk 7]
```diff
@@ -6,4 +6,97 @@
...
+    border-radius: 0.5rem;
+}
+
+nav a:active {
+    border-color: rgba(150, 49, 49, 0.524);
+
+}
+
+form {
+    display: flex;
```
Comment (iampava) [inline style.css]: Some styles here can be removed by using utility classes. See comment above.

**Review by iampava — CHANGES_REQUESTED** [review]
Overall looks good, but can be improved.

File: index.html (hunk 1 above)
Comment (damdiana) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (damdiana) [reply index.html:5]: done

File: style.css [hunk 8]
```diff
@@ -6,4 +6,97 @@
...
+}
+
+/* nav bar */
+
+.nav {
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+nav ul {
```
Comment (iampava) [inline style.css]: There's no `ul` left in the page. Delete this rule.

File: style.css [hunk 9]
```diff
@@ -6,4 +6,97 @@
...
+    flex-flow: row wrap;
+    gap: 2rem;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+nav a {
+    text-decoration: none;
+    display: block;
```
Comment (iampava) [inline style.css]: I think you can remove this line.

File: style.css [hunk 10]
```diff
@@ -6,4 +6,97 @@
...
+
+nav a:hover {
+    background-color: rgba(150, 49, 49, 0.524);
+    border-radius: 0.5rem;
+
+}
+
+nav a:active {
+    border-color: rgba(150, 49, 49, 0.524);
+
```
Comment (iampava) [inline style.css]: Let's remove this newline.

File: style.css [hunk 11]
```diff
@@ -6,4 +6,97 @@
...
+    font-size: inherit;
+    position: absolute;
+    left: 0.5em;
+    width: 1em;
+    bottom: 0.65em;
+    fill: var(--red);
+
+}
+
+.flex {
```
Comment (iampava) [inline style.css]: Let's extract these util classes into a `utils.css` file.

**Review by iampava — CHANGES_REQUESTED** [review]

**Review by iampava — APPROVED** [review]
Looks great! Let's squash and merge!
